Task_DEV_5/Task_DEV_5/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Do we need this file in the repo? Looks like it is some visual studio file.

Reply:
Done.